doc/services/sensing/index.rst

```diff
@@ -0,0 +1,282 @@
+    * ``Virtual sensor``: relies on other sensor(s), ``physical`` or ``virtual``, focus on
+                          data fusion.
+
+* Data Driven Model
+    * ``Polling mode``: periodical sampling rate
+    * ``Interrupt mode``: data ready, threshold interrupt etc.
+
+* Scheduling
+    * single thread main loop for all sensor objects sampling and process.
+
+* Buffer Mode for Batching
+* Configurable Via Device Tree
+
+API Design
+**********
+
+API Organization
+================
+
+* Sensing Subsystem
+    * Sensor Types (:zephyr_file:`include/zephyr/sensing/sensing_sensor_types.h`)
+    * Data Types (:zephyr_file:`include/zephyr/sensing/sensing_datatypes.h`)
+    * Sensing Subsystem API (:zephyr_file:`include/zephyr/sensing/sensing.h`)
+    * Sensing API (:zephyr_file:`include/zephyr/sensing/sensing_sensor.h`)
+
+Below diagram shows the API position and scope:
+
+.. image:: images/sensing_api_org.png
+   :align: center
+   :alt: Sensing subsystem API organization.
+
+``Sensing Subsystem API`` is for Applications.
+``Sensing Sensor API`` is for development ``sensors``.
+
+
+Major Flows
+=========================
+
+* Sensor Configuration Flow
+
+.. image:: images/sensor_config_flow.png
+   :align: center
+   :alt: Sensor Configuration Flow (App set report interval to hinge angel sensor example).
+
+* Sensor Data Flow
+
+.. image:: images/sensor_data_flow.png
+   :align: center
+   :alt: Sensor Data Flow (App receive hinge angel data through data event callback example).
+
+Sensor Types And Instance
+=========================
+
+The ``Sensing Subsystem`` supports multiple instances of the same sensor type,
+there're two methods for Applications to identify and open an unique sensor instance:
+
+* Enumerate all sensor instances
+
+  :c:func:`sensing_get_sensors` returns all current board configuration supported sensor instances'
+  information in a :c:struct:`sensing_sensor_info` pointer array .
+
+  Then Applications can use :c:func:`sensing_open_sensor` to
+  open specific sensor instance for future accessing, configuration and receive sensor data etc.
+
+  This method is suitable for supporting some up-layer frameworks like ``CHRE``, ``HID`` which need
+  dynamic enumerate underlying platforms' sensor instances.
+
+* Open the sensor instance by devicetree node directly
+
+  Applications can use :c:func:`sensing_open_sensor_by_dt` to open a sensor instance directly with
+  sensor devicetree node name.
+
+  For example:
+
+.. code-block:: c
+
+   sensing_open_sensor_by_dt(DEVICE_DT_GET(DT_NODELABLE(base_accel)), cb_list, handle);
+   sensing_open_sensor_by_dt(DEVICE_DT_GET(DT_CHOSEN(zephyr_sensing_base_accel)), cb_list, handle);
+
+This method is useful and easy use for some simple Application which just want to access specific
+  sensor(s).
+
+
+``Sensor type`` follows the
+    `HID standard sensor types definition <https://usb.org/sites/default/files/hutrr39b_0.pdf>`_.
```

Comment:
Same here, the extra spaces on the line continuation cause a rendering error.
```suggestion
``Sensor type`` follows the
`HID standard sensor types definition <https://usb.org/sites/default/files/hutrr39b_0.pdf>`_.
```

Reply:
Fixed, thanks a lot!